zzz_generated/tv-casting-app/zap-generated/IMClusterCommandHandler.cpp

```diff
@@ -1309,15 +1309,6 @@ void DispatchServerCommand(CommandHandler * apCommandObj, const ConcreteCommandP
             }
             break;
         }
-        case Commands::TestSpecific::Id: {
-            Commands::TestSpecific::DecodableType commandData;
-            TLVError = DataModel::Decode(aDataTlv, commandData);
-            if (TLVError == CHIP_NO_ERROR)
-            {
-                wasHandled = emberAfTestClusterClusterTestSpecificCallback(apCommandObj, aCommandPath, commandData);
-            }
-            break;
-        }
```

Comment:
I wonder why this is removed ?

Reply:
It seems correct that this was removed because I see that the test cluster server is only enabled in EP1. When I go to the command settings there I see that the incoming for "TestSpecific" command is disabled so it should not have been generated for before. The old helpers for incoming commands must have been outdated and were generating randomly. It should not do that with the new template changes in the commandHandler template. However I have to warn you that there is still one use case which has not been dealt with yet where you may have different command or attribute settings for the same cluster in different endpoints for eg the attribute default value. That is a complex situation and needs to be looked at in terms of the generated code and embedded code changes. I do not think we have hit that situation yet but still a possibility that a user may hit this.